dojo/tools/hcl_appscan/parser.py

```diff
@@ -24,60 +33,85 @@ def get_findings(self, file, test):
+                            title = self.xmltreehelper(item)
+                            description = description + "Issue-Type-Name:" + title + "\n"
                         case 'location':
-                            location = item.text
-                            description = description + "Location: " + location + "\n"
+                            location = self.xmltreehelper(item)
+                            description = description + "Location:" + location + "\n"
                         case 'domain':
-                            domain = item.text
-                            description = description + "Domain: " + domain + "\n"
+                            domain = self.xmltreehelper(item)
+                            title += "_" + domain
+                            description = description + "Domain:" + domain + "\n"
+                        case 'threat-class':
+                            threatclass = self.xmltreehelper(item)
+                            description = description + "Threat-Class:" + threatclass + "\n"
+                        case 'entity':
+                            entity = self.xmltreehelper(item)
+                            title += "_" + entity
+                            description = description + "Entity:" + entity + "\n"
+                        case 'security-risks':
+                            security_risks = self.xmltreehelper(item)
+                            description = description + "Security-Risks:" + security_risks + "\n"
+                        case 'cause-id':
+                            causeid = self.xmltreehelper(item)
+                            title += "_" + causeid
+                            description = description + "Cause-Id:" + causeid + "\n"
+                        case 'url-name':
+                            urlname = self.xmltreehelper(item)
+                            title += "_" + urlname
+                            description = description + "Url-Name:" + urlname + "\n"
                         case 'element':
-                            element = item.text
-                            description = description + "Element: " + element + "\n"
+                            element = self.xmltreehelper(item)
+                            description = description + "Element:" + element + "\n"
                         case 'element-type':
-                            elementtype = item.text
-                            description = description + "ElementType: " + elementtype + "\n"
+                            elementtype = self.xmltreehelper(item)
+                            description = description + "ElementType:" + elementtype + "\n"
                         case 'path':
-                            path = item.text
-                            description = description + "Path: " + path + "\n"
+                            path = self.xmltreehelper(item)
+                            title += "_" + path
```

Comment:
Same comment as above

```suggestion
                            title += "_" + path.strip()
```